subgraph/tests/utils.ts

```diff
@@ -162,7 +168,12 @@ export function createNewBuildEvent(
     new ethereum.EventParam('token', ethereum.Value.fromUnsignedBigInt(token))
   );
   newBuildEvent.parameters.push(
-    new ethereum.EventParam('commitHash', ethereum.Value.fromString(commitHash))
+    new ethereum.EventParam(
+      'commitHash',
+      ethereum.Value.fromBytes(
```

Comment:
Here is where the magic happens: Since the event parameter is seen as "bytes", the trick is to actually create a Bytes value. The naming of the function "fromBytes" as a bit confusing but it also sets the type of the value to "BYTES":

```
    static fromBytes(bytes: Bytes): Value {
      return new Value(ValueKind.BYTES, changetype<u32>(bytes))
    }
```

As seen here: https://github.com/graphprotocol/graph-ts/blob/main/chain/ethereum.ts#L319